backend/br_restore.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Set basic runtime settings
+echo "Restoring from env: $1"
+echo "Restoring backup type: $2"
+echo "Restoring backup dated: $3"
+export PATH=/home/vcap/deps/0/apt/usr/lib/postgresql/15/bin:$PATH
+date=$(date '+%Y-%m-%d-%H%M')
+prefix="$date"_"$2"_"$1"
+log_file="$prefix"_restore_log.txt
+prefix="$3"_"$2"_"$1"
```

Comment:
Why are we redefining `prefix` here?

Reply:
Looks like left-over code. Good catch. Will do another commit this afternoon.